src/Core/Ripgrep.re

```diff
@@ -83,33 +83,26 @@ and dispose = unit => unit;
 module RipgrepProcessingJob = {
   type pendingWork = {
     callback: list(string) => unit,
-    bytes: list(Bytes.t),
+    queue: Queue.t(Bytes.t) // WARNING: mutable data structure
```

Comment:
Cool! I hadn't used `Queue.t` from the stdlib much... interesting to see it in action.

Reply:
First time I've used it too!